iis_service_action.ps1

```diff
@@ -0,0 +1,62 @@
+Param(
+    [parameter(Mandatory = $true)]
+    [ValidateSet( 'start', 'stop', 'restart')]
```

Comment:
Same comment as other repos, is this case sensitive?  If so you probably need to make a note of it in action.yml since your script relies on it being lowercase.